x-pack/plugins/infra/public/components/asset_details/asset_details.tsx

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import {
+  EuiFlyout,
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiFlyoutBody,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiTabs,
+  EuiTab,
+  useEuiTheme,
+} from '@elastic/eui';
+import { css } from '@emotion/react';
+import Metadata from './metadata/metadata';
+import { LinkToUptime } from './links/link_to_uptime';
+import { LinkToApmServices } from './links/link_to_apm_services';
+import { Processes } from './processes/processes';
+import type { HostNodeRow } from './types';
+import type { InventoryItemType } from '../../../common/inventory_models/types';
+import type { SetNewHostFlyoutOpen } from '../../pages/metrics/hosts/hooks/use_host_flyout_open_url_state';
+
+export enum FlyoutTabIds {
+  METADATA = 'metadata',
+  PROCESSES = 'processes',
+}
+
+export type TabIds = 'metadata' | 'processes';
+
+export interface Tab {
+  id: FlyoutTabIds;
+  name: any;
```

Comment:
Thank you for spotting that! Fixed ( That surprised me - I copied the suggested type and the translation was returned as any instead of string, now I see where it came from 😅 )